src/EnergyPlus/WindowManager.hh

```diff
@@ -804,94 +644,13 @@ struct WindowManagerData : BaseGlobalStruct
         this->tmpReflectSolBeamBack = 0.0;
         this->tmpReflectVisBeamFront = 0.0;
         this->tmpReflectVisBeamBack = 0.0;
-        //            this->DbgTheta = Array1D<Real64>(11, {0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 82.5, 89.5});
-        //            this->DbgTSol = Array1D<Real64>(11, 0.0);
-        //            this->DbgRbSol = Array1D<Real64>(11, 0.0);
-        //            this->DbgTVis = Array1D<Real64>(11, 0.0);
-        //            this->DbgFtAbs = Array2D<Real64>(5, 11, 0.0);
-        //            this->DbgBkAbs = Array2D<Real64>(5, 11, 0.0);
-        //            this->DbgFTAbsDiff = Array1D<Real64>(5, 0.0);
-        //            this->DbgBkAbsDiff = Array1D<Real64>(5, 0.0);
     }
 
     // Default Constructor
     WindowManagerData()
-        : sigma(5.6697e-8), TKelvin(DataGlobalConstants::KelvinConv), nume(107), numt3(81), gcon(3, 5, 5, 0.0), gvis(3, 5, 5, 0.0), gcp(3, 5, 5, 0.0),
-          gwght(5, 5, 0.0), gfract(5, 5, 0.0), gnmix(5, 0), gap(5, 0.0), thick(5, 0.0), scon(5, 0.0), tir(10, 0.0), emis(10, 0.0), rir(10, 0.0),
-          AbsRadGlassFace(10, 0.0), thetas(10, 0.0), thetasPrev(10, 0.0), fvec(10, 0.0), fjac(10, 10, 0.0), dtheta(5, 0.0), zir(10, 10, 0.0),
-          ziri(10, 10, 0.0), ddeldt(10, 10, 0.0), dtddel(10, 10, 0.0), qf(10, 0.0), hf(10, 0.0), der(5, 10, 0.0), dhf(5, 10, 0.0), sour(10, 0.0),
-          delta(5, 0.0), hcgap(5, 0.0), hrgap(5, 0.0), rgap(6, 0.0), rs(6, 0.0), arhs(6, 0.0), MaxNumOfIncidentAngles(20),
-          MaxSpectralDataElements(800)
     {
-        AirProps.allocate(8);
```

Comment:
So good!

Reply:
Yeah!  Like 60 ArrayND objects converted to std arrays.  Hopefully this puts a dent into the number of ArrayND objects where N > 1 so that we can reduce our dependency on Objexx for multidimensional arrays.